examples/seq_reader.py

```diff
@@ -10,7 +10,7 @@
 
 data = nutchpy.sequence_reader.slice(5,20,path)
 # print(data)
-assert len(data) == 2
+assert len(data) == 3
```

Comment:
Since the example data has 8 records `[0..7]`, the `slice(5, 20)` function returns last three records `[5,6,7]`.